.github/workflows/build.yml

```diff
@@ -109,44 +73,16 @@ jobs:
         path: ${{github.workspace}}/suscan/suscan-tools_0.3.0-${{steps.vars.outputs.sha_short}}_amd64.deb
 
   build-macos:
-    needs: dep_target
     runs-on: macos-latest
     steps:
     - name: Install dependencies
-      run: |
-        # Add custom homebrew repos
-        brew tap pothosware/homebrew-pothos
-        brew update
-        # Install deps
-        # soapyaudio removed from due to deprecated hamlib API
-        # soapyosmo  removed because of random compilation errors
-        # soapyairspyhf  removed because of random compilation errors
-        brew install --force --overwrite python@3.12
-        brew install libsndfile volk fftw soapysdr libxml2 portaudio json-c
-        python3 -Im pip install setuptools
-        # sudo rm /usr/local/bin/2to3* /usr/local/bin/idle3* /usr/local/bin/pydoc3*
-        brew install soapyrtlsdr soapyhackrf soapybladerf soapyairspy soapyredpitaya soapyiris limesuite soapyplutosdr
-        brew install --head soapyuhd
-        # TODO: needed?
-        #sudo mkdir -p /usr/local/lib/SoapySDR/modules0.8
-        #sudo chmod -R a+rw /usr/local/lib/SoapySDR/modules0.8
-        # SDRPlay API
-        wget https://www.sdrplay.com/software/SDRplay_RSP_API-MacOSX-3.07.3.pkg
-        sudo installer -pkg SDRplay_RSP_API-MacOSX-3.07.3.pkg -target /
-        # SoapySDRPlay3 from source
-        git clone https://github.com/pothosware/SoapySDRPlay3
-        cd SoapySDRPlay3
-        cmake -DCMAKE_BUILD_TYPE=Release -B build .
-        cmake --build build
-        cd build
-        sudo make install
-        cd ../..
+      run: brew install libsndfile volk fftw soapysdr
```

Comment:
Isn't json-c missing here?

Reply:
It compiles. Either something is wrong in cmake or it is not needed... I will check later...

Reply:
It compiles, but the resulting binary doesn't have SigMF support. The json-c dependency is optional but we need to test it here nonetheless.

Reply:
That was an easy fix :)